crates/bitwarden-exporters/src/cxf/login.rs

```diff
@@ -67,23 +57,18 @@ pub(super) fn to_login(
 impl From<Login> for BasicAuthCredential {
     fn from(login: Login) -> Self {
         BasicAuthCredential {
-            urls: login
-                .login_uris
-                .into_iter()
-                .flat_map(|uri| uri.uri)
-                .collect(),
-            username: login.username.map(|value| EditableField {
-                id: random_id(),
-                field_type: FieldType::String,
-                value,
-                label: None,
-            }),
-            password: login.password.map(|value| EditableField {
-                id: random_id(),
-                field_type: FieldType::ConcealedString,
-                value,
-                label: None,
-            }),
+            urls: vec![],
```

Comment:
If I understand this correctly, the urls have been moved from the `BasicAuthCredential` to the `CredentialScope` right? If so, why is there a `urls` field in `BasicAuthCredential` still?